illumio_consts.py

```diff
@@ -21,8 +21,9 @@
 # and limitations under the License.
 
 PROTOCOL_LIST = ["tcp", "udp"]
-ENFORCEMENT_MODE_LIST = ["idle", "visibility_only", "full", "selective"]
+ENFORCEMENT_MODE_LIST = ["idle", "visibility_only", "full", "selective", None]
```

Comment:
Don't add None to these lists. Just do the falsy check separately:

```
if enforcement_mode and enforcement_mode not in ENFORCEMENT_MODE_LIST:
    ...
```

Note that the `illumio` lib has an `EnforcementMode` enum as well that you can use:

```
if enforcement_mode and enforcement_mode not in illumio.EnforcementMode:
    ...
```

Reply:
Updated constant list value and code